cpp/bench/ann/src/common/benchmark.hpp

```diff
@@ -483,7 +483,14 @@ void register_search(std::shared_ptr<const dataset<T>> dataset,
                   ->MeasureProcessCPUTime()
                   ->UseRealTime();
 
-      if (metric_objective == Mode::kThroughput) { b->ThreadRange(threads[0], threads[1]); }
+      if (metric_objective == Mode::kThroughput) {
```

Comment:
Let's not put algorithm-specific stuff in the common files. I agree with @achirkin about this one. If needed, you could propoagate this down to the specific algorithms, but the common stuff should be agnostic of he algoithmmic specific.

Reply:
Thanks, @cjnolet for spotting this. @tarang-jain , could you please move the warning into the faiss-related header (perhaps into the set_search_params)? There, you should be able to query the `cuvs::bench::benchmark_n_threads > 1` from `common/util.hpp`.

Reply:
Fixed this! Marking as resolved.